launch-rstudio-container.slurm

```diff
@@ -53,22 +53,23 @@ touch ${workdir}/singularity_vars
 # https://github.com/rstudio/rstudio/blob/v1.4.1106/src/cpp/server/ServerSessionManager.cpp#L126
 export SINGULARITYENV_RSTUDIO_SESSION_TIMEOUT=0
 
+export SINGULARITYENV_USER=$(id -un)
+
```

Comment:
We had `USER` and `SINGULARITYENV_USER` with the same value. This consolidates them to one environment variable.